apps/src/applab/CrosshairOverlay.js

```diff
@@ -125,6 +154,17 @@ CrosshairOverlay.prototype.create_ = function () {
 
   this.text_ = document.createElementNS(SVG_NS, 'text');
   this.ownElement_.appendChild(this.text_);
+
+  this.elementIdBubble_ = document.createElementNS(SVG_NS, 'rect');
+  this.elementIdBubble_.setAttribute('width', TEXT_RECT_WIDTH);
+  this.elementIdBubble_.setAttribute('height', TEXT_RECT_HEIGHT);
+  this.elementIdBubble_.setAttribute('rx', TEXT_RECT_RADIUS);
+  this.elementIdBubble_.setAttribute('ry', TEXT_RECT_RADIUS);
```

Comment:
This code is duplicated from above.   Please extract a small helper, something like `createTextBubbleElement_`.